bhagavad_gita_api/config.py

```diff
@@ -4,6 +4,19 @@
 from pydantic import AnyUrl, BaseSettings
 
 
```

Comment:
Can you please import and load dotenv? 
Currently, it doesn't seem to be able to load the values from `.env`.

```python
from dotenv import load_dotenv

load_dotenv()
```